payment/controllers.go

```diff
@@ -50,6 +52,13 @@ func CredentialRouter(service *Service) chi.Router {
 	return r
 }
 
+// PaymentRouter handles calls relating to payments
+func PaymentRouter(service *Service) chi.Router {
+	r := chi.NewRouter()
+	r.Method("POST", "/stripe/webhooks", middleware.InstrumentHandler("HandleStripeWebhook", HandleStripeWebhook(service)))
```

Comment:
does it make sense to have webhooks be the base, just so we don't end up sprinkling implementations around as we add more?

Reply:
+1, I like `/v1/payments/webhooks/stripe` better too.